DigitalLearningSolutions.Data/Models/MultiPageFormData/AddNewCentreCourse/CourseContentData.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace DigitalLearningSolutions.Data.Models.MultiPageFormData.AddNewCentreCourse
+{
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models;
+
+    public class CourseContentData
+    {
+        public CourseContentData(){}
+
+        public CourseContentData(
+            IEnumerable<Section> availableSections,
+            bool includeAllSections,
+            IEnumerable<int>? selectedSectionIds
+        )
+        {
+            AvailableSections = availableSections;
+            IncludeAllSections = includeAllSections;
+            SelectedSectionIds = selectedSectionIds;
+        }
+
+        public bool IncludeAllSections { get; set; }
+
+        public IEnumerable<Section> AvailableSections { get; set; }
+
+        [Required(ErrorMessage = "You must select at least one section")]
+        public IEnumerable<int>? SelectedSectionIds { get; set; }

```

Comment:
Removing the `set` from any of these MultiPageFormData/..../...Data classes will cause the Json conversion to fail since it need to use them to set the values.